meetings/2025-10-02.md

```diff
@@ -0,0 +1,147 @@
+- Jesse: At the ODI we are working on education and training project. How do you convice SME and org to use Solid. For SME, a good argument in favor of using Solid is they don't have to worry about the compliance issue . Would like to work with Luke to work on this issue ( convincing to use Solid)
+- WP : With business, is more simple if they don't have to manage the data. [...]
+- WP: In the economics of decentralised data and trading data the wallet is a great solution for that.
+- Luke: A big issue is that ontology internally is all over the place, ppl have hard time to believe they can be one unique source of truth coming from the user
+- WP: will be great if redhat provide some infrastructure, as it is currently very embrionic
+- JZ: Another important part of Solid is interoperability
+- Tom Byrd: Another good selling point: [...]
+- Marc Haddle: When it come to patient data: way better if have it in one place. Can be challenging to maintain the link of user data ( because of de-auth ). As long as data is hosted on someone else's server, you don't have control.
+- Luke: Can be interesting for user centric AI-agent . ( Ai is currently a good selling point)
+- WP: AI has been a good selling point in our side. If redhat really want to differentiate itself from others, this can be a good point.
+- J: One our side onthology has been a good branding, as it create AI ready data. Is RH working on onthology ?
+- L: Low on the priority. The improvement from the onthology is obscure because the functionnality currently works. GDPR is not easy but working. I try to make a point on working on a small thing where Solid would work well.
+- JZ: with redhat you get apache, why don't you get Solid too? wouldn't cost much for RedHat, and users could start playing with it.
+- L: Yes and that's why we talked about keycloak, this is what we use by default.
+- J: Pretty sure Inrupt makes Solid work with Keycloak but closed source
+- FT: Would be very valuable to have this work. Be great to convice Tim to release this code
+
+### Announcements, current topics relevant for the Group (please add your points!)
+
+- JZ: there is a number of things discussed at the ODI advisory commitee:
+
+#### use cases
+
+- ODI is currently deciding which use cases to support.
+- J: clarification: we are not going to directly fund those use cases because of a limitation of the grant. We are building a framework to see which use cases should get support in priority. Call for action is to put Practitionners efforts into the Catalogue. Please reach to us if you need help or have any questions.
```

Comment:
```suggestion
- J: clarification: we are not going to directly fund use cases because of a limitation of the grant. We are building a framework to see which use cases should get support in priority. Call for action is to put Practitioners' efforts into the Catalogue. Please reach out to us if you need help or have any questions.
```